src/actions/playground.ts

```diff
@@ -0,0 +1,27 @@
+import { Action, ActionCreator } from 'redux'
+
+/**
+ * The `type` attribute for an `Action` which updates the `IPlaygroundState`
+ * `editorValue`
+ */
+export const UPDATE_EDITOR_VALUE: string = 'UPDATE_EDITOR_VALUE'
+
+/**
+ * Represents an `Action` which updates the `editorValue` of a
+ * `IPlaygroundState`
+ * @property type           - Unique string identifier for this `Action`
+ * @property newEditorValue - The new string value for `editorValue`
+ */
+export interface IUpdateEditorValue extends Action {
+  type: string
+  newEditorValue: string
```

Comment:
You should follow the Redux FSA(Flux Standard Action) convention. The payload of the action should be under the name `payload`. A payload is typically an object in complex action, but in this case I think a string suffice.

Reply:
To clarify, do you mean the naming convention only? Or do you mean that we should keep the`payload` object as an arbitrarily typed object (like in most example online)? I feel that the typing should remain strict for a particular payload.